cmd/lotus-bench/main.go

```diff
@@ -407,16 +407,26 @@ var sealBenchCmd = &cli.Command{
 			}
 			verifyWinningPost2 := time.Now()
 
+			ppt, err := sealedSectors[0].SealProof.RegisteredWindowPoStProof()
```

Comment:
Did we consider adding nv to RegisteredWindowPoStProof?

Reply:
Yes. The problem is that that method is called by old versions of specs-actors, which would then need to be refactored, and...

Reply:
Oh, yeah..